src/providers/generalWalletProvider.tsx

```diff
@@ -82,6 +82,17 @@ export const GeneralWalletContext = createContext<IGeneralWalletContext>({
 	isOnEVM: false,
 });
 
+const getSolanaProvider = () => {
```

Comment:
```suggestion
const getPhantomSolanaProvider = () => {
```
As its functionality